spec/adapter_spec.rb

```diff
@@ -29,4 +29,44 @@
       expect(subject.image.current_path).to eq file_path("../uploads", "file.jpg")
     end
   end
+
+  describe ".mount_base64_uploaders" do
+    let(:uploader) { Class.new CarrierWave::Uploader::Base }
+
+    subject do
+      Post.mount_base64_uploaders(:images, uploader)
+      Post.new
+    end
+
+    it "handles normal file uploads" do
+      sham_rack_app = ShamRack.at('www.example.com').stub
+      sham_rack_app.register_resource("/test.jpg", file_path("fixtures", "test.jpg"), "images/jpg")
+      sham_rack_app.register_resource("/test2.jpg", file_path("fixtures", "test.jpg"), "images/jpg")
+      subject[:images] = ["test.jpg", "test2.jpg"]
+      subject.save!
+      subject.reload
+      p subject
```

Comment:
Do you consider removing this line? This seems to me debugging related, should be removed before release. Or use Logger instead.